pkgs/data/fonts/siji/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "siji-${date}";
+  date = "2016-05-14";
+
+  src = fetchgit {
```

Comment:
There is no release-specific code. For any revision of siji, you can download a tarball using the link `https://github.com/stark/siji/archive/<rev>.tar.gz`.

Try:
```
rasen@ashmalko ~ % nix-prefetch-url https://github.com/stark/siji/archive/95369afac3e661cb6d3329ade5219992c88688c1.tar.gz                                                          
downloading ‘https://github.com/stark/siji/archive/95369afac3e661cb6d3329ade5219992c88688c1.tar.gz’... [0/0 KiB, 0.0 Ki [0/0 KiB, 0.0 KiB/s]
path is ‘/nix/store/1lmsk8y7mvlcvxq6rxxz2pnx6jjg01ad-95369afac3e661cb6d3329ade5219992c88688c1.tar.gz’
1pz73pl2ylbhdbikh2nrl39xfyc8am71rhqpzx8lr28slz025q8m
```